clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -519,7 +519,6 @@ AlterConfigsResult incrementalAlterConfigs(Map<ConfigResource,
      *
      * @param replicaAssignment     The replicas with their log directory absolute path
      * @return                      The AlterReplicaLogDirsResult
-     * @throws InterruptedException Interrupted while joining I/O thread
```

Comment:
@dongjinleekr Could you please take a look at this ```throws```? It was added by your PR (https://github.com/apache/kafka/pull/7083).

Reply:
@chia7712 Okay. Let me see. :smiley:

Reply:
@chia7712 `InterruptedException` is not thrown anymore; This method does not call `KafkaFutureImpl#get` method anymore. It's safe to remove it.